### DISCUSSION TITLE
No recovery key and no passkey

### DISCUSSION THREAD

**Discussion by aleksaingh** [ente-io/ente#4085] [Q&A]
I tried to log in to my ente account with my password but since I got a new phone there was no passkeys on the device and I already wiped the old phone which had the passkeys. Please help

**Comment by ua741** [comment]
Please write to support [at] ente.

**Comment by aleksaingh** [comment]
I know my password to my ente account but when I switched phones I didn't
have the passkey nor the recovery key
